packages/cli/core/src/commands/org/resources/pull.ts

```diff
@@ -218,7 +218,7 @@ export default class Pull extends Command {
     const flags = await this.getFlags();
     return flags.model
       ? flags.model.resourcesToExport
-      : buildResourcesToExport(flags.resourceTypes);
+      : buildResourcesToExport(flags.resourceTypes!);
```

Comment:
Why the Non-null assertion operator?
If not specified by the user, flags.resourceTypes should be an empty array